### PR TITLE
SH 388 Update Group API with member add/edit/delete

### PR DESCRIPTION
**Pull request by reshmi-nair**
This API is to update group details by providing group id. Also member add/edit/delete can be performed.
Code coverage:
group-actors : missing instructions/branches : 84%:53%
group-service : missing instructions/branches : 89%:69%

### CODE REVIEW

File: service/test/controllers/CreateGroupControllerTest.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import java.util.Map;
 import javax.ws.rs.core.Response;
 import org.junit.Before;
+import org.junit.Ignore;
```
Comment (Hari-stackroute) [inline service/test/controllers/CreateGroupControllerTest.java:11]: It is not used, can be removed.

File: group-actors/src/main/java/org/sunbird/service/impl/MemberServiceImpl.java [hunk 2]
```diff
@@ -32,13 +36,62 @@ public static MemberService getInstance() {
 
   @Override
   public Response addMembers(List<Member> member) throws BaseException {
+    member.forEach(m -> m.setStatus(JsonKey.ACTIVE));
```
Comment (Hari-stackroute) [inline group-actors/src/main/java/org/sunbird/service/impl/MemberServiceImpl.java:39]: please check to set all the operations in one foreach loop

Comment (reshmi-nair) [reply group-actors/src/main/java/org/sunbird/service/impl/MemberServiceImpl.java]: Done

File: service/test/controllers/CreateGroupControllerTest.java (hunk 1 above)
Comment (reshmi-nair) [reply service/test/controllers/CreateGroupControllerTest.java]: Done

**Pull request merged**